src/EditorFeatures/CSharp/Highlighting/KeywordHighlighters/AsyncAwaitHighlighter.cs

```diff
@@ -10,9 +11,20 @@
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.KeywordHighlighting.KeywordHighlighters
 {
-    internal abstract class AbstractAsyncHighlighter<TNode> : AbstractKeywordHighlighter<TNode> where TNode : SyntaxNode
+    [ExportHighlighter(LanguageNames.CSharp)]
+    internal class AsyncAwaitHighlighter : AbstractKeywordHighlighter
     {
-        protected void HighlightRelatedKeywords(SyntaxNode node, List<TextSpan> spans)
+        protected override bool HighlightNode(SyntaxNode node)
```

Comment:
alternative name could be: IsHighlightableNode

Reply:
Honestly, I like the first option better.

Reply:
IsHighlightableNode?